### ISSUE TITLE
how to use batch normalization layer without model.fit?

### ISSUE THREAD

**Issue by lfz** [tflearn/tflearn#310]
Hi, I wrote a network in tflearn, and I want to optimize it with tensorflow optimizer, but I found that the batch normalization layer didn't work, I found the update function for batch norm is `update_mean_var`, but I don't know how to use it? Thank you!

**Comment by aymericdamien** [comment]
You have to use `tflearn.is_training(True/False, session=...)` to indicate if you are training or testing. See: http://tflearn.org/config/#is_training.

Something like that:
```
net = ... # A network built with tflearn layers
tflearn.is_training(True, session=sess)
sess.run(train_op)
tflearn.is_training(False, session=sess)
sess.run([summary_op, test_op])
```

**Comment by lfz** [comment]
Thank you!
